railties/lib/rails/generators/app_base.rb

```diff
@@ -280,6 +280,7 @@ def gem_for_database
         case options[:database]
         when "mysql"          then ["mysql2", [">= 0.3.18", "< 0.5"]]
         when "postgresql"     then ["pg", ["~> 0.18"]]
+        when "sqlserver"      then ["tiny_tds", nil]
```

Comment:
If possible, can we have this on the same line as the other `sqlserver` dependency specified on line 286? Or at least on the line above / below?